x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/LifecycleExecutionStateTests.java

```diff
@@ -71,6 +71,69 @@ public void testEqualsAndHashcode() {
             LifecycleExecutionStateTests::mutate);
     }
 
+    public void testGetCurrentStepKey() {
```

Comment:
Since the cases are so small, I think it's okay to leave them as a single test. We could always split them up if we needed to in the future. As it is this test was just moved from a different location verbatim (rather than me writing it).